selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -277,7 +276,19 @@ void WifiManager::stateChange(unsigned int new_state, unsigned int previous_stat
 
 // https://developer.gnome.org/NetworkManager/stable/gdbus-org.freedesktop.NetworkManager.Device.Wireless.html
 void WifiManager::propertyChange(const QString &interface, const QVariantMap &props, const QStringList &invalidated_props) {
-  if (interface == NM_DBUS_INTERFACE_DEVICE_WIRELESS && props.contains("LastScan")) {
+  if (props.contains("ActiveConnection")) {
+    QString path = props.value("ActiveConnection").value<QDBusObjectPath>().path();
+    if (emptyPath(path)) {
+      setCurrentSsid("");
```

Comment:
this is true when `StateChanged` emits state 110 (`NM_DEVICE_STATE_DEACTIVATING`) - https://developer-old.gnome.org/NetworkManager/stable/nm-dbus-types.html#NMState